homeassistant/components/comfoair/sensor.py

```diff
@@ -0,0 +1,154 @@
+"""Platform to control a Zehnder ComfoAir 350 ventilation unit."""
+
+import logging
+
+from bitstring import BitArray
+
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from . import DOMAIN, SIGNAL_COMFOAIR_UPDATE_RECEIVED, ComfoAirModule
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_COMFORT_TEMPERATURE = "comfort_temperature"
+ATTR_OUTSIDE_TEMPERATURE = "outside_temperature"
+ATTR_SUPPLY_TEMPERATURE = "supply_temperature"
+ATTR_RETURN_TEMPERATURE = "return_temperature"
+ATTR_EXHAUST_TEMPERATURE = "exhaust_temperature"
+ATTR_AIR_FLOW_SUPPLY = "air_flow_supply"
+ATTR_AIR_FLOW_EXHAUST = "air_flow_exhaust"
+ATTR_FAN_SPEED_MODE = "speed_mode"
+
+SENSOR_TYPES = {
+    ATTR_COMFORT_TEMPERATURE: [
+        "Comfort Temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        0xD2,
+        0 * 8,
+        8,
+    ],
+    ATTR_OUTSIDE_TEMPERATURE: [
+        "Outside Temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        0xD2,
+        1 * 8,
+        8,
+    ],
+    ATTR_SUPPLY_TEMPERATURE: [
+        "Supply Temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        0xD2,
+        2 * 8,
+        8,
+    ],
+    ATTR_RETURN_TEMPERATURE: [
+        "Return Temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        0xD2,
+        3 * 8,
+        8,
+    ],
+    ATTR_EXHAUST_TEMPERATURE: [
+        "Exhaust Temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        0xD2,
+        4 * 8,
+        8,
+    ],
+    ATTR_AIR_FLOW_EXHAUST: ["Exhaust airflow", "%", "mdi:fan", 0xCE, 6 * 8, 8],
+    ATTR_AIR_FLOW_SUPPLY: ["Supply airflow", "%", "mdi:fan", 0xCE, 7 * 8, 8],
+    ATTR_FAN_SPEED_MODE: ["Speed mode", "", "mdi:fan", 0xCE, 8 * 8, 8],
+}
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the ComfoAir sensor platform."""
+    unit = hass.data[DOMAIN]
+
+    sensors = []
+    for resource in SENSOR_TYPES.keys():
```

Comment:
Iterating a dict, iterates the keys by default.
```suggestion
    for resource in SENSOR_TYPES:
```